tests/epyccel/test_arrays_multiple_assignments.py

```diff
@@ -8,7 +8,7 @@
                                     ARRAY_DEFINITION_IN_LOOP,
                                     INCOMPATIBLE_REDEFINITION_STACK_ARRAY,
                                     STACK_ARRAY_DEFINITION_IN_LOOP,
-                                    ASSIGN_ARRAYS_ONE_ANOTHER, ARRAY_ALREADY_IN_USE)
+                                    ASSIGN_ARRAYS_ONE_ANOTHER, ARRAY_ALREADY_IN_USE, STACK_ARRAY_NON_LITERAL_SHAPE)
```

Comment:
We should now import `STACK_ARRAY_UNKNOWN_SHAPE`

Reply:
Fixed [e11207b](https://github.com/pyccel/pyccel/pull/993/commits/e11207bc7ceaa38e20e92a9d1d4b8a53920e77cb)